examples/websocket-chat-application/websocket_chat_application.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates a simple WebSocket chat application using Ballerina.
```

Comment:
Edit: This BBE demonstrates a simple WebSocket chat application using Ballerina.